main.go

```diff
@@ -139,12 +140,18 @@ func getConnWithRetry(flags *flagStorage) (c *gcsx.Connection, err error) {
 ////////////////////////////////////////////////////////////////////////
 
 // Mount the file system according to arguments in the supplied context.
```

Comment:
shouldn't this comment be for mountWithArgs method?

Reply:
fixed